chapters/fa/chapter2/2.mdx

```diff
@@ -301,7 +301,7 @@ outputs = model(**inputs)
 </div>
 
 {:else}
-برای این مثال، نیازمند مدلی با سر مخصوص دسته‌بندی رشته‌ها(برای تشخیص منفی یا مثبت بودن جملات) هستیم. پس به جای کلاس `TFAutoModel` از کلاس `TFAutoModelForSequenceClassification` استفاده می‌کنیم:
+برای این مثال، نیازمند مدلی با سَر مخصوص دسته‌بندی رشته‌ها(برای تشخیص منفی یا مثبت بودن جملات) هستیم. پس به جای کلاس `TFAutoModel` از کلاس `TFAutoModelForSequenceClassification` استفاده می‌کنیم:
```

Comment:
There must be a space before parentheses openings.

Reply:
Fixed.